lib/download.go

```diff
@@ -9,15 +9,100 @@ import (
 	"strings"
 )
 
-// DownloadFromURL : Downloads the binary from the source url
-func DownloadFromURL(installLocation string, url string) (string, error) {
+var (
+	PubKeyId     = "72D7468F"
+	PubKeyPrefix = "hashicorp_"
+	PubKeyUri    = "https://www.hashicorp.com/.well-known/pgp-key.txt"
+)
```

Comment:
What do you think of consolidating default values at https://github.com/warrensbox/terraform-switcher/blob/master/lib/defaults.go or nearby the https://github.com/warrensbox/terraform-switcher/blob/master/main.go#L40-L51 ?